django/contrib/auth/forms.py

```diff
@@ -154,14 +154,14 @@ def validate_passwords(
         if not usable_password:
             return self.cleaned_data
 
-        if not password1:
+        if password1 == "":
             error = ValidationError(
                 self.fields[password1_field_name].error_messages["required"],
                 code="required",
             )
             self.add_error(password1_field_name, error)
 
-        if not password2:
+        if password2 == "":
```

Comment:
If this line is undone, no test fail! :scream:
Could you please ensure that there are assertions for this logic change?

Reply:
@nessita That would be another three test cases. OK?

Reply:
Yes! The more the merrier?

Reply:
Actually, it also works by extending the single test per form. 👍